.gitignore

```diff
@@ -76,3 +76,6 @@ target/
 
 # Vim
 *.sw*
+
```

Comment:
can you remove this? you can create a `.gitignore` locally without including it in this repo's `.gitignore`. Instructions here: https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore